unblob/processing.py

```diff
@@ -87,6 +88,10 @@ class ExtractionConfig:
     entropy_plot: bool = False
     max_depth: int = DEFAULT_DEPTH
     skip_magic: Iterable[str] = DEFAULT_SKIP_MAGIC
+    skip_extension: Iterable[str] = attr.field(
+        default=DEFAULT_SKIP_EXTENSION,
+        converter=lambda value: [extension for extension in value if extension],
+    )
```

Comment:
Why are empty extensions removed?
```suggestion
    skip_extension: Iterable[str] = DEFAULT_SKIP_EXTENSION
```

Reply:
Providing an empty extension would mean skipping all files without an extension, does it make sense from a usage perspective ?

Reply:
did some thinking, I'm not against it. Applying.